CMake file sets incorrect `INTERFACE_INCLUDE_DIRECTORIES` path

This was brought to my attention here https://github.com/PCSX2/pcsx2/issues/12819 (who was using an [AUR build](https://aur.archlinux.org/packages/plutosvg-git) of plutosvg from git)

Building plutosvg commit eb34f711def70e068f9307c81b4359a8443f6728 with the following CMake args;
```
-DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX="E:\Git\TheLastRar\pcsx2\deps" -DBUILD_SHARED_LIBS=ON -DPLUTOSVG_ENABLE_FREETYPE=ON -DPLUTOSVG_BUILD_EXAMPLES=OFF -B build -G Ninja
```

Will setup a `plutosvgTargets.cmake` file that creates a `plutosvg::plutosvg` target like so;
```
set_target_properties(plutosvg::plutosvg PROPERTIES
  INTERFACE_COMPILE_DEFINITIONS "PLUTOSVG_HAS_FREETYPE"
  INTERFACE_INCLUDE_DIRECTORIES "/plutosvg"
  INTERFACE_LINK_LIBRARIES "plutovg::plutovg;Freetype::Freetype"
)
```
This differs from release 0.0.7, which created a `plutosvg::plutosvg` target like so;
```
set_target_properties(plutosvg::plutosvg PROPERTIES
  INTERFACE_COMPILE_DEFINITIONS "PLUTOSVG_HAS_FREETYPE"
  INTERFACE_INCLUDE_DIRECTORIES "${_IMPORT_PREFIX}/include/plutosvg"
  INTERFACE_LINK_LIBRARIES "plutovg::plutovg;Freetype::Freetype"
)
```

When attempting to use said target from the git build (via `target_link_libraries(imgui PUBLIC plutosvg::plutosvg)`), I get the following error

```
CMake Error in 3rdparty/imgui/CMakeLists.txt:
  Imported target "plutosvg::plutosvg" includes non-existent path
    "/plutosvg"
  in its INTERFACE_INCLUDE_DIRECTORIES.  Possible reasons include:
  * The path was deleted, renamed, or moved to another location.
  * An install or uninstall procedure did not complete successfully.
  * The installation package was faulty and references files it does not
  provide.
CMake Generate step failed.  Build files cannot be regenerated correctly.
-- Generating done (0.3s)
```

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -20,6 +20,9 @@ set(plutosvg_sources
 add_library(plutosvg ${plutosvg_sources})
 add_library(plutosvg::plutosvg ALIAS plutosvg)
 
+include(GNUInstallDirs)
+include(CMakePackageConfigHelpers)
+
 set_target_properties(plutosvg PROPERTIES
     SOVERSION ${PLUTOSVG_VERSION_MAJOR}
     C_VISIBILITY_PRESET hidden
@@ -54,9 +57,6 @@ if(PLUTOSVG_ENABLE_FREETYPE)
     target_compile_definitions(plutosvg PUBLIC PLUTOSVG_HAS_FREETYPE)
     target_link_libraries(plutosvg PUBLIC Freetype::Freetype)
 endif()
-
-include(GNUInstallDirs)
-include(CMakePackageConfigHelpers)
 
 configure_package_config_file(
     "${CMAKE_CURRENT_SOURCE_DIR}/cmake/plutosvgConfig.cmake.in"

